.ci_support/linux_64_.yaml

```diff
@@ -6,3 +6,9 @@ channel_targets:
 - conda-forge main
 docker_image:
 - quay.io/condaforge/linux-anvil-cos7-x86_64
+pin_run_as_build:
+  python:
+    min_pin: x.x
+    max_pin: x.x
+python:
+- 3.9.* *_cpython
```

Comment:
Do we need this? What's this do?

Reply:
We do not need this p sure, it was updated from conda-smithy locally and accidentally committed

Reply:
Removing in the next commit